ingest/bin/infer-dengue-serotype.py

```diff
@@ -0,0 +1,52 @@
+#! /usr/bin/env python3
+
+import argparse
+import json
+from sys import stdin, stdout
+
+def parse_args():
+    parser = argparse.ArgumentParser(
+        description="Dengue specific processing of metadata, infer serotype from virus_tax_id",
+        formatter_class=argparse.ArgumentDefaultsHelpFormatter,
+    )
+    parser.add_argument(
+        "--virus-tax-id",
+        type=str,
+        default="virus_tax_id",
+        help="Column name containing the NCBI taxon id of the virus serotype.",
+    )
+    parser.add_argument(
+        "--out-col",
+        type=str,
+        default="ncbi_serotype",
+        help="Column name to store the inferred serotype.",
+    )
+    return parser.parse_args()
+
+
+def _set_dengue_serotype(record, col="virus_tax_id"):
```

Comment:
_non-blocking_

Slight nit-pick to rename this to `get_dengue_serotype` because `set_*` usually implies that it will update the record.

Reply:
Thanks, fixed in https://github.com/nextstrain/dengue/pull/20/commits/18536b412bf1d46f40b5a5d510144fd6dc79df11